mne/inverse_sparse/tests/test_mxne_inverse.py

```diff
@@ -334,3 +334,28 @@ def test_split_gof_meg(forward, idx, weights):
     want = 100 * want / want.sum()
     assert_allclose(gof_split, want, atol=1e-3, rtol=1e-2)
     assert_allclose(gof_split.sum(), 100, rtol=1e-5)
+
+
+@testing.requires_testing_data
+@pytest.mark.slowtest
+@pytest.mark.parametrize('condition', [
+    'Left Auditory',
+    'Right Auditory',
+    'Left visual',
+    'Right visual'
+])
+@pytest.mark.parametrize('loose', [0, 0.9])
+def test_mxne_inverse_sure(forward, condition, loose):
+    """Test the SURE criterion for automatic alpha selection on MEG data."""
+    noise_cov = mne.read_cov(fname_cov)
```

Comment:
I would rather use a simulated dataset where you know what is the true number of sources you want to find.

Reply:
Ok so no tests on MEG data?

Reply:
simulated MEG is enough to validate that it works and you know the ground truth